pages/linux/kdocker.md

```diff
@@ -0,0 +1,28 @@
+# kdocker
+
+> Easily dock applications to system tray.
+> More information: <https://github.com/user-none/KDocker>.
+
+- Click on a window and send it to system tray:
+
+`kdocker`
+
+- Open an application and send it to system tray:
+
+`kdocker {{application}}`
+
+- Send focused windows to system tray:
```

Comment:
How do you focus more than 1 window at the same time?
```suggestion
- Send focused window to system tray:
```
Note: This command only works with terminals because you are in the terminal when you press enter :upside_down_face:

Reply:
Seems to work with my VScode